tool/gravity/cli/rpcagent.go

```diff
@@ -315,16 +331,44 @@ func deployAgents(ctx context.Context, env *localenv.LocalEnvironment, req deplo
 
 // newDeployAgentsRequest creates a new request to deploy agents on the local cluster
 func newDeployAgentsRequest(ctx context.Context, env *localenv.LocalEnvironment, req deployAgentsRequest) (*rpc.DeployAgentsRequest, error) {
-	servers, err := verifyCluster(ctx, req)
-	if err != nil {
-		return nil, trace.Wrap(err)
+	// If hostname is specified in the request, deploy agent only on specified node
+	var servers []rpc.DeployServer
+	if req.hostname != "" {
+		server, err := findServerByHostname(req.clusterState, req.hostname)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+
+		verifiedServer := rpc.NewDeployServer(server)
+		if err := verifyNode(ctx, verifiedServer, req.proxy); err != nil {
+			return nil, trace.Wrap(err)
+		}
+
+		servers = append(servers, verifiedServer)
+	} else {
+		verifiedServers, err := verifyCluster(ctx, req)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+		servers = append(servers, verifiedServers...)
+	}
+
+	// If version is not specified in the request, use the current build version
+	if req.version == "" {
+		req.version = strings.Split(version.Get().Version, "+")[0]
```

Comment:
What's "split by +" for?

Reply:
https://github.com/gravitational/gravity/blob/46b9ee065b121ed99ac593cb98084ef079ac020b/tool/gravity/cli/rpcagent.go#L416
Copied over from `getGravityPackage` func. It doesn't seem like we need to split version by '+'. I guess we had a different version format in the past?

Reply:
@a-palchikov Do you recall by any chance why we had this "split by +"? I remember it was needed for something but can't remember why.

Reply:
It might have had something to do with semver parsing but I need to go through the history to know more/remember.